pkg/cmd/dev.go

```diff
@@ -256,13 +294,31 @@ var devCommand = cli.Command{
 			Aliases: []string{"config"},
 			Usage:   "Path to Stainless config file",
 		},
+		&cli.StringFlag{
+			Name:    "branch",
+			Aliases: []string{"b"},
+			Usage:   "Which branch to use",
+		},
+		&cli.StringSliceFlag{
+			Name:    "target",
+			Aliases: []string{"t"},
+			Usage:   "The target build language(s)",
+		},
+		&cli.BoolFlag{
+			Name:    "watch",
+			Aliases: []string{"w"},
+			Value:   false,
+			Usage:   "Run in 'watch' mode to loop and rebuild when files change.",
+		},
 	},
-	Action: func(ctx context.Context, cmd *cli.Command) error {
-		return runDevMode(ctx, cmd)
-	},
+	Action: runPreview,
 }
 
-func runDevMode(ctx context.Context, cmd *cli.Command) error {
+func runPreview(ctx context.Context, cmd *cli.Command) error {
+	// Clear the screen and move the cursor to the top
+	fmt.Print("\033[2J\033[H")
+	os.Stdout.Sync()
```

Comment:
q: thoughts on only doing this when `--watch`? don't feel super strongly either way personally

Reply:
Oh, yes, that should have been the behavior. I moved things around and changed that unintentionally.